tests/conftest.py

```diff
@@ -30,7 +31,36 @@ async def websocket():
 
 
 @pytest_asyncio.fixture
```

Comment:
Just moving context a bit earlier in the file to make the order a bit more logical.